moveit_core/robot_model/include/moveit/robot_model/robot_model.h

```diff
@@ -53,6 +52,7 @@
 
 #include <Eigen/Geometry>
 #include <iostream>
+#include <moveit/logging/logging.h>
```

Comment:
seems to me this include should only be in the .cpp file

Reply:
outdated